code/code/misc/statistics.cc

```diff
@@ -207,6 +209,10 @@ int init_game_stats(void)
       vlogf(LOG_BUG, "bad value for rate of change of equipment load potential");
     }
 
+    if (fscanf(fp, "%lf\n", &stats.burnrate) != 1) {
```

Comment:
Why not store it in DB? You could reuse the `configuration` table, it's a key-value mapping -- or even better, create a new table for all stats. See `code/sys/migrations.cc` for a comfy way to do that.

Reply:
This makes sense.  I was trying to keep my first commit simple so I just copied the stats.load format.  There are many magic constants scattered around that could be pulled into this.  I'll look into it.